package.json

```diff
@@ -39,8 +39,8 @@
         "pre-commit": "npm run build:lib:types && concurrently lint-staged npm:test"
     },
     "peerDependencies": {
-        "react": "17.0.2",
-        "react-dom": "17.0.2"
+        "react": "^18.3.1",
```

Comment:
We should actually lock these down - please remove the `^` for these two

Reply:
Done